h/api/search/transform.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+"""Functions for transforming data for or from the search index."""
+
+import copy
+
+from h.api import uri
+
+
+def prepare(annotation):
+    """
+    Prepare the given annotation for indexing.
+
+    Scan the passed annotation for any target URIs or document metadata URIs
+    and add normalised versions of these to the document.
+    """
+
+    # FIXME: When this becomes simply part of a search indexing operation, this
+    # should probably not mutate its argument.
+    _normalise_annotation_target_uris(annotation)
+
+
+def render(annotation):
+    """
+    Render an annotation retrieved from search for public display.
+
+    Receives data direct from the search index and reformats it for rendering
+    or display in the public API. At the moment all this does is remove
```

Comment:
I would remove "At the moment all this does..." since that's a comment that is liable to get out of date and it's more detail than anyone needs to know here. Want to know exactly what it does? Go look at that function.